packages/react/src/components/Grid/Grid.js

```diff
@@ -0,0 +1,311 @@
+/**
+ * Copyright IBM Corp. 2019
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import { settings } from 'carbon-components';
+import { breakpoints } from '@carbon/layout';
```

Comment:
I think we'll need to make sure we add this to `react/package.json` 👍

Reply:
oops, missed this one originally. resolved in 35d8769.